### DISCUSSION TITLE
Use Google sheet as manual data source

### DISCUSSION THREAD

**Discussion by swaggerino** [ghostfolio/ghostfolio#1210] [Q&A]
I was wondering how I can use google sheets as manual data source for the history of stock prices. I think I've set everything up correctly over at the google dev console (kinda followed along here https://theoephraim.github.io/node-google-spreadsheet/#/getting-started/authentication) and browsing the ghostfolio code it seems to look for sheets name just like the symbol. However on import (json) I get the error that google sheets is not a valid data source for the symbol...

Other than that I'm wondering what the sheet has to look like. Column A = date and B = price?

Thanks

**Comment by dtslvr** [comment]
The data source `GOOGLE_SHEETS` is experimental. My tests have revealed that the response time is unfortunately too long for practical use.

**Comment by Marx2** [comment]
I have such a sheet and simply FIle->Download->as CSV, than I'm trying to import it. Having it fully automated would be nice addition
